pgjdbc/src/test/java/org/postgresql/test/jdbc2/BaseTest4.java

```diff
@@ -149,4 +149,8 @@ protected void assertBinaryForSend(int oid, boolean expected, Supplier<String> m
     assertEquals(message.get() + ", useBinaryForSend(oid=" + oid + ")", expected,
         con.unwrap(BaseConnection.class).getQueryExecutor().useBinaryForSend(oid));
   }
+
+  public  void assumeExtensionInstalled(String extension) throws SQLException {
```

Comment:
```suggestion
  public void assumeExtensionInstalled(String extension) throws SQLException {
```

double space look odd